main.go

```diff
@@ -21,6 +23,7 @@ import (
 
 func init() {
 	rand.Seed(time.Now().UnixNano())
+	jsoniter.RegisterExtension(&jsonutil.RawMessageExtension{})
```

Comment:
Do we want to register it for all of the jsoniter library, or can/should it be scoped to the `jsonConfigValidationOn` and `jsonConfigValidationOff` configs?